docs/blog/2019-08-13-Localised-strings-from-Kentico-Cloud-GatsbyJS-GraphQL/index.md

```diff
@@ -0,0 +1,96 @@
+
+With your content items in Kentico Cloud, you should be able to see them when you conduct a GraphQL query.
+
+Here is an example of the GraphQL query you will use to retrieve the ID and CodeName for the Content Items from Kentico Cloud using GatsbyJS. Think of this as the 'key' you would need to retrieve the 'value'. You can see the 'codename' and 'id' as potential options that you could use in the below GraphQL query.
+![A view of the GraphiQL preview of retrieving language nodes](images/GraphiQL-retrieve-lang-nodes.png)
+
+You can retrieve all localization strings using the following GraphQL query.
+![Retrieving all language versions of the localized strings](images/GraphiQL-retrieve-lang-variants.png)
+
+However, to retrieve the actual 'value', there’s a little more to do in your GraphQL query.
+
+Here is an example of a GraphQL query where I am using the 'CodeName'.
+![GraphiQL retrieve language nodes localized strings condition code name](images/GraphiQL-retrieve-lang-variants-based-on-condition-codename.png)
+
+And, here is an example of a GraphQL query where I am using the ID.
+![GraphiQL retrieve language nodes localized strings condition ID](images/GraphiQL-retrieve-lang-variants-based-on-condition-id.png)
+
+As you can see, once you know which 'key' you need, then it's pretty simple to get the 'value' and also detect the language variant you need to retrieve in GatsbyJS using this GraphQL query format.
+
+## Separate projects
+
+Before we go any further, it is important to take stock of your localization string content items. Creating culture versions of each item doesn’t increase your number of Content Items so you’re unlikely to be jumping up through plans. However, the number of localization string Content Items is something to keep in mind. Also, consider the number of environments/projects you have in your subscription, which could multiply the number of Content Items you have in total.
+
+We would always advocate pragmatism when creating your content models. In addition, maybe consider creating the localization strings as a separate project that feeds into all the projects that need to utilise them. This is a solution that has worked across several of our projects and ensures that all localized strings are available to all projects and all environments, removing the overhead for updates.
+
+## Summary - A recap of the steps involved in creating localized content in Kentico Cloud
+
+There’s a lot to consider there so let's recap:
+
+* Create a Content Type for localise string items.
+* Add appropriate Content Items with the values for each localized string item.
+* Create the relevant culture alternatives for the content items created.
+* Once you know the desired culture you would then look in GraphQL nodes for the localized strings you need to show and use the filter option to help you bring specific localized strings back.
+* Utilise the query within your GatsbyJS layout/React code.
```

Comment:
```suggestion
* Utilize the query within your GatsbyJS layout/React code.
```

Reply:
@gillkyle thanks for this. Hopefully, I've amended the few that were missed. 

I've also run prettier and this should also come through.